app/views/news_tematica/news_tematicas/elegir_contenidos.html.haml

```diff
@@ -36,10 +41,13 @@
   &nbsp; &nbsp;
   = submit_tag('Generar HTML', class: 'boton_reborde_naranja_grande')
 
-:javascript
-  jQuery(function($) {
-    $('.js_ver_resto_titulares').click(function() {
-      $('.js_resto_titulares').slideDown();
-      $(this).hide();
-    });
-  });
+:coffee
```

Comment:
I prefer to have the JS on separate files. But I just was maintaining the current style. I'll move it to a coffee file ;)